processor/src/advice/merkle_set.rs

```diff
@@ -0,0 +1,102 @@
+use super::{MerkleError, MerklePath, MerklePathSet, MerkleTree, NodeIndex, SimpleSmt, Vec, Word};
+
+// ADVICE SET
+// ================================================================================================
```

Comment:
minor nit: This should probably be `MERKLE SET`

Reply:
Done! Updated also @ docs